src/apps/chifra/internal/names/output_test.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2021 The TrueBlocks Authors. All rights reserved.
```

Comment:
I love that you always add GoLang tests. It's so important. I wanted to recognize it. Thanks.